cogboard-webapp/src/components/widgets/types/ZabbixWidget/index.js

```diff
@@ -48,9 +51,9 @@ const ZabbixWidget = ({ id, lastvalue }) => {
 
   const calculatePercentageValue = () => {
     if (!lastvalue) return 0;
-    if (!checkMetricHasMaxValue()) return parseInt(lastvalue, 10);
+    if (!checkMetricHasMaxValue) return parseInt(lastvalue, 10);
 
-    return convertToBytes(lastvalue);
+    return Math.round((100 * lastvalue) / (maxValue * Math.pow(10, 9)));

```

Comment:
I like previous version better - with named function

Reply:
Now is correct. This line is for calculating the percentage value and function is called calculatePercentageValue already.